tasks/update-granules-cmr-metadata-file-links/index.js

```diff
@@ -89,6 +98,7 @@ async function updateGranulesCmrMetadataFileLinks(event) {
     .map(({ name, type }) => [name, type]));
 
   const cmrGranuleUrlType = get(config, 'cmrGranuleUrlType', 'both');
+  const excludeFileRegexPattern = get(config, 'excludeFileRegex', null);
```

Comment:
Why do we need to default to null? If we change it to:
`const excludeFileRegexPattern = get(config, 'excludeFileRegex');`
We can still do the check
`if (excludeFileRegexPattern)`

Reply:
https://github.com/nasa/cumulus/pull/3791/commits/369d4bde6d68e5678f7f6af3726859eda9fc7b9f
changed